### ISSUE TITLE
Replace "Select all lines" with "Copy to clipboard"

### ISSUE THREAD

**Issue by wilzbach** [adamdruppe/adrdox#19]
![image](https://user-images.githubusercontent.com/4370550/36637752-6f576970-19e3-11e8-90c7-2ff0e0a0f9a6.png)

FYI: it's possible to directly copy to the user clipboard (it's not a nice API, but well). There are things like [clipboard.js](https://github.com/zenorocha/clipboard.js) (which is pretty bloated imho), because essentially all you need to do is `.select()` it and call `execCommand('copy').
Here's the trick I use for run.dlang.io:

```js
// Programmatic copy-to-clipboard
// No one ever said that JavaScript is sane
function copyText(text) {
	var fakeElement = document.createElement('textarea');
	// Move element out of screen vertically
	fakeElement.style.position = 'fixed';
	fakeElement.style.top = '100%';
	fakeElement.readOnly = true;
	fakeElement.value = text;
	document.body.appendChild(fakeElement);

	fakeElement.select();
	document.execCommand('copy');

	document.body.removeChild(fakeElement);
}
```

**Comment by adamdruppe** [comment]
Yeah, I can play with that. It actually should just take the call to document.execCommand after selecting all - see the code here: https://github.com/adamdruppe/adrdox/commit/60dff3d919a200d8f3003d72447eea230c0e2b0f

worked on my firefox, haven't tried other browsers yet.

**Comment by adamdruppe** [comment]
closing as the feature works now, passes my tests.
